### DISCUSSION TITLE
should we keep reclassify in the lib ?

### DISCUSSION THREAD

**Discussion by 12rambau** [12rambau/sepal_ui#693] [General]
I use the planet connector all the time, and most of the functions as well. We developped the reclassify module in the lib for the sdg15.3.1 and the montain green index applications but in the end I never use it. Do you think it really worth the shot keeping it in the lib instead of an independent module ? 

pros: 
- easy to integrate 
- input agnostic 

cons:
- heavy (it has a negative impact on nearly everything when we load the lib
- badly tested (I'm not sure it still work out of the box)

**Comment by dfguerrerom** [comment]
Yes, you're right I think there is no great advantage to have it in the package... 
I would prefer to have it as separate tool, but then, its integration into different apps would be more complicated... but... we can create, a single `reclassify_model.py`, and multiple `reclassify_tile` and `view`, so one of then can be used in the main module and the others with different purposes.
